build.py

```diff
@@ -382,13 +382,11 @@ def pytorch_cmake_args(images):
 
 def onnxruntime_cmake_args(images, library_paths):
     cargs = [
+        '-DTRITON_ENABLE_ONNXRUNTIME_TENSORRT=ON',
```

Comment:
We should hold off on enabling TRT support for jetpack till I add in the updated ORT build (1.8.1) which enables TRT 8 support.